web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/ys_views_basic.services.yml

```diff
@@ -0,0 +1,5 @@
+services:
+  # Service for managing views basic plugins.
+  plugin.manager.views_basic:
```

Comment:
Just an issue with the naming convention. This work is not actually a plugin manager. Maybe we will create a custom plugin at some point, but for now it's just a service.

Reply:
Gotcha, thanks for the direction on this one. Changed.